README.md

```diff
@@ -53,110 +32,14 @@
 [![gzip size](http://img.badgesize.io/https://unpkg.com/preact/dist/preact.min.js?compression=gzip)](https://unpkg.com/preact/dist/preact.min.js)
 [![install size](https://packagephobia.now.sh/badge?p=preact)](https://packagephobia.now.sh/result?p=preact)
```

Comment:
Can I remove this badge, it's not that informative

Reply:
oh, yeah, definitely.